.github/workflows/functional-tests.yml

```
@@ -117,7 +117,7 @@ jobs:
       - name: Set current date as env variable
         run: echo "NOW=$(date +'%Y-%m-%dT%H-%M-%S')" >> $GITHUB_ENV
       - name: Sanitize branch name
-        run: echo "BRANCH_SANITIZED=$(echo ${{ env.BRANCH_NAME }} | tr "/" "_")" >> $GITHUB_ENV
+        run: echo "BRANCH_SANITIZED=$(echo ${{ env.BRANCH_NAME }} | tr '/()'' '_')" >> $GITHUB_ENV
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC1009:info:1:56: The mentioned syntax error was in this simple command

(shellcheck)